.gitignore

```
@@ -1,5 +1,4 @@
-logs/
-.idea/
 __pycache__/
 .ipynb_checkpoints/
 *.h5py
+venv
```

* Since virtual environment is created in the top directory only, absolute path can be used instead.
* Downloaded pickles should be ignored by Git too.

```suggestion
/data/*.pickle
/venv
```

---

will update it thanks!!